extensions/attributegen/config.pb.html

```diff
@@ -184,7 +184,7 @@ <h2 id="Match">Match</h2>
 expression</a>
 that may use <a href="https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/rbac_filter#condition">builtin attributes</a>.</p>
 
-<p>Example:
+<p>{{<tabset category-name="example">}} Example:
```

Comment:
please make this change in extensions/attributegen/config.proto

And then run 🏃‍♀️  “ make extensions-docs”, it will update the html file.
You need to have protoc installed and have protoc-gen-docs from istio.io/tools repo available in path.

Please let me know if you would rather I do it.

Reply:
Thanks for the directions. Hopefully this is better. I'm not sure where the `extensions/metadata_exchange/declare_property.pb.html` came in but was part of the `make extensions-docs`.